### PR TITLE
Add CAP class

### LINKED ISSUE #94: Add CAP Class

### PR DESCRIPTION
**Pull request by maryjess**
- Create Cap class
- Modify Person to support CAP field
- Update UI to reflect Cap field

Resolves #94.

### CODE REVIEW

**Comment by maryjess** [comment]
Do you guys have any comments on whether I should capitalise the `"CAP"` in the UI and JSON file, or should I just keep it as `"Cap"`? Currently it is the latter in the JSON file, and the former in the UI.

**Review by JasonCP14 — CHANGES_REQUESTED** [review]
EditPersonDescriptorTest.java need to include CAP

**Review by lulucopter — APPROVED** [review]
Overall LGTM, good job

File: src/test/java/seedu/address/model/person/CapTest.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import org.junit.jupiter.api.Test;
+
+class CapTest {
+    // null parameters of Cap() will be tested upon parsing Cap values, as double cannot be null.
+    @Test
+    void isValidCap() {
+        // invalid cap
+        assertFalse(Cap.isValidCap(-5.0, 5.0)); // negative cap value
```
Comment (lulucopter) [inline src/test/java/seedu/address/model/person/CapTest.java:13]: Minor, but recommended to add in tests for alphabetical inputs and symbols eg. "asd" and "!@#"

**Review by JasonCP14 — APPROVED** [review]

**Pull request merged**